spec/models/card_spec.rb

```diff
@@ -14,6 +14,6 @@
   end
 
   it "should set right default review date" do
-    expect(card.review_date.round(0)).to eq(Time.now.utc.round(0) + 3.days)
+    expect(card.review_date).to eq(Time.now.to_date + 3.days)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.